AllJoyn/Platform/Bridge.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+# Visual Studio 15
```

Comment:
But it opens in VS 14 as well. So no issue as such.

---
In reply to: [75178273](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75178273) [](ancestors = 75178273,75173869)